shuup/reports/settings.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2021, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+
+# report
+DEFAULT_REPORTS_ITEM_LIMIT = 2000
```

Comment:
Add a good documentation about what this setting is for and add missing blank line at the end of the file